tests/convergence/bgp_convergence_config.py

```diff
@@ -1,97 +1,91 @@
-import pytest
+# import pytest
```

Comment:
better add `@pytest.mark.skip` with proper comment rather than comment-out entire test. So, we will easily track it.